regression-test/suites/query_p0/sql_functions/math_functions/test_math_unary_always_nullable.groovy

```diff
@@ -82,4 +82,20 @@ suite("test_math_unary_alway_nullable") {
         select rowid, dsqrt(val), dsqrt(val) is null from test_math_unary_alway_nullable order by rowid;
     """
 
+    qt_is_nan_1 """
```

Comment:
the test is too simple.

Reply:
add regression like https://github.com/apache/doris/pull/40462